src/components/notifications/Discord.vue

```diff
@@ -16,4 +16,46 @@
         <label for="discord-prefix-message" class="form-label">{{ $t("Prefix Custom Message") }}</label>
         <input id="discord-prefix-message" v-model="$parent.notification.discordPrefixMessage" type="text" class="form-control" autocomplete="false" :placeholder="$t('Hello @everyone is...')">
     </div>
+
+    <div class="mb-3">
+        <label for="discord-message-type" class="form-label">{{ $t("Select message type") }}</label>
+        <br>
+        <select id="discord-message-type" v-model="$parent.notification.discordChannelType" class="form-select">
+            <option value="channel">{{ $t("Send to channel") }}</option>
+            <option value="createNewForumPost">{{ $t("Create new forum post") }}</option>
+            <option value="postToThread">{{ $t("Post to existing thread / forum post") }}</option>
+        </select>
+    </div>
+
+    <div v-if="$parent.notification.discordChannelType === 'createNewForumPost'">
+        <div class="mb-3">
+            <label for="discord-target" class="form-label">{{ $t("Forum post name") }}</label>
+            <input id="discord-target" v-model="$parent.notification.postName" type="text" class="form-control" autocomplete="false">
+            <div class="form-text">
+                {{ $t("whatHappensAtForumPost") }}"{{ $t("Post to existing thread / forum post") }}"
```

Comment:
Please consider using the same pattern as for `discordThreadID` ^^
There are likely languages which have different word orders (f.ex. RTL languages)